src/redux/services/clusterResourceWatcher.ts

```diff
@@ -113,15 +121,18 @@ const watchResource = async (
             await watchResource(dispatch, handler, kubeConfig, previewResources, handler.kindPlural);
           }
         }
+        resourcesToUpdate.push(resource);
         // dispatch(updateClusterResource(resource));
         return;
       }
       if (type === 'MODIFIED') {
+        resourcesToUpdate.push(resource);
         // dispatch(updateClusterResource(resource));
         return;
       }
       if (type === 'DELETED') {
-        dispatch(deleteClusterResource(resource));
```

Comment:
We should remove `deleteClusteResource` and `updateClusterResource` actions from main reducer as they are not used anymore and remove the comments from here